projects/lf-documentation/src/app/app.component.css

```diff
@@ -2,10 +2,14 @@ button {
     margin-right: 5px;
 }
 .arrow-left {
-  background-color: white;
-  border-width: 1px;
-  border-style: solid;
-  border-color: #e0e0e0;
+  vertical-align: sub;
+  color: rgba(0,0,0,.5);
+}
```

Comment:
add some highlight on hover

Reply:
at least for the actual icon, not for the button unless it's as big as the square which it doesn't look like it is